### ISSUE TITLE
empty player array on csgo

### ISSUE THREAD

**Issue by yumerchalashkan** [Austinb/GameQ#555]
Hello. I'm trying to get players' information from my csgo server. player array comes blank. All other information of the server comes correct. Why is the player array blank?

`

  | 7.3.12Array
-- | --
  | (
  | [45.10.56.63:27015] => Array
  | (
  | [dedicated] => d
  | [game_descr] => SonJeton Jailbreak
  | [game_dir] => csgo
  | [game_id] => 730
  | [gq_address] => 45.10.56.63
  | [gq_dedicated] => d
  | [gq_gametype] => SonJeton Jailbreak
  | [gq_hostname] => [TR] SonJeton JailBreak Sunucusu \| ★ Türkler Geliyor ! \| CS.
  | [gq_joinlink] => steam://connect/45.10.56.63:27015/
  | [gq_mapname] => jb_buyukisyan_sonjetonnew
  | [gq_maxplayers] => 64
  | [gq_mod] => csgo
  | [gq_name] => Counter-Strike: Global Offensive
  | [gq_numplayers] => 33
  | [gq_online] => 1
  | [gq_password] => 0
  | [gq_port_client] => 27015
  | [gq_port_query] => 27015
  | [gq_protocol] => source
  | [gq_transport] => udp
  | [gq_type] => csgo
  | [hostname] => [TR] SonJeton JailBreak Sunucusu \| ★ Türkler Geliyor ! \| CS.
  | [keywords] => CS.Center,TR,awp,buyukisyan,eldiven,fun,fun,hediye,jail,jail,jailbreak,jailbreak,jb,nametag,oyunhavuzu,pro,public,son
  | [map] => jb_buyukisyan_sonjetonnew
  | [max_players] => 64
  | [num_bots] => 0
  | [num_players] => 33
  | [os] => l
  | [password] => 0
  | [players] => Array
  | (
  | )
  |  
  | [port] => 27015
  | [protocol] => 17
  | [secure] => 1
  | [steam_id] => 85568392923311922
  | [steamappid] => 730
  | [teams] => Array
  | (
  | )
  |  
  | [version] => 1.37.6.8
  | )
  |  
  | )
  |  

`

**Comment by Gasjki** [comment]
Hey @ocaliskan ! Check this URL for more information: [GameTracker forum](https://www.gametracker.com/games/csgo/forum.php?thread=91691)

TL;DR: CSGO servers need special configuration in order to make players information available.
Here's the config:

`host_players_show 2`
